web/client/utils/cesium/Layers.js

```diff
@@ -35,9 +35,8 @@ var Layers = {
     updateLayer: function(type, layer, newOptions, oldOptions, map) {
         var layerCreator = layerTypes[type];
         if (layerCreator && layerCreator.update) {
-            return layerCreator.update(layer, newOptions, oldOptions, map);
+            return layerCreator.update(newOptions, oldOptions, map);
```

Comment:
You cannot change the signature of update, other layers (e.g. MarkerLayer) expect a layer as the first parameter